setup.py

```diff
@@ -0,0 +1,34 @@
+"""
+    botwo package
+
+"""
+
+from setuptools import setup, find_packages
```

Comment:
Can we separate all publishing/installation related stuff to a different PR?

Reply:
Done